vignettes/desctable.Rmd

```diff
@@ -369,8 +366,8 @@ In the *tests* arguments, you can also provide function definitions, functional
 iris %>%
   group_by(Species) %>%
   desctable(tests = list(.auto = tests_auto,
-                         Sepal.Width = ~function(f) oneway.test(f, var.equal = F),
-                         Petal.Length = ~. %>% oneway.test(var.equal = T),
+                         Sepal.Width = ~oneway.test(., var.equal = F),
+                         Petal.Length = ~oneway.test(.,var.equal = T),
                          Sepal.Length = ~purrr::partial(oneway.test, var.equal = T))) %>%
```

Comment:
~oneway.test(., var.equal = T)

Those examples are redundant and should be discarded